examples/stable_diffusion_v2/docs/en/controlnet.md

```diff
@@ -137,6 +163,15 @@ Here are samples generated by a bird image with DeeplabV3Plus segmentation edge
   <em> Generated Images with ControlNet on DeeplabV3Plus segmentation edge maps </em>
 </p>
 
+<div align="center">
+<img src="https://github.com/Gaohan123/mindone/assets/20148503/6d543d0b-e1c2-447b-805a-19d9253a488b" width="160" height="240" />
+<img src="https://github.com/Gaohan123/mindone/assets/20148503/90835ad9-38aa-4ca2-862a-0344c0760463" width="160" height="240" />
+<img src="https://github.com/Gaohan123/mindone/assets/20148503/bf1bc4e9-c16c-4d37-8b72-cbc83fd8569e" width="160" height="240" />
+</div>
+<p align="center">
+  <em> Generated Images with ControlNet on DeeplabV3Plus segmentation edge maps </em>
+</p>
+
```

Comment:
It would be greate to include some generated images using the openpose control as well.

Reply:
I will complete this part.